### DISCUSSION TITLE
Integración SINCO M&E y SINCO SGD - Registro y consulta de expedientes

### DISCUSSION THREAD

**Discussion by NestorFGG** [SINCOSOFT-SAS/SGD#38] [General]
Esta integración permitirá registrar una carpeta de archivo a partir de un equipo creado, así sobre esta, registrar y consultar documentos como llegar al SGD los archivos de OT y Preoperacionales.

**Comment by NestorFGG** [comment]
Se finaliza Fase 1 de desarrollo por parte de SINCO SGD "Configuración de integración"

**Comment by aleja550** [comment]
Este desarrollo se realizo por parte de SGD pero MYE no finalizo la parte que realizaran, y debido a cosmos han detenido desarrollos.
